code/modules/recycling/disposal-construction.dm

```diff
@@ -19,7 +19,14 @@
 	var/dpdir = 0	// directions as disposalpipe
 	var/base_state = "pipe-s"
 
-	// update iconstate and dpdir due to dir and type
+/obj/structure/disposalconstruct/examine(mob/user)
+	..()
+	if(anchored)
+		to_chat(user, "<span class='info'>It's bolted down to the floor plating.</span>")
+	else
+		to_chat(user, "<span class='info'>It has been detached from the floor plating.</span>")
```

Comment:
"Has been" implies that it was attached at some point, which isn't necessarily true.